src/app/static/src/tests/components/downloadResults/downloadResults.test.ts

```diff
@@ -36,7 +36,7 @@ describe("Download Results component", () => {
         jest.useRealTimers()
     })
 
-    const createStore = (hasUploadPermission = true, getUserCanUpload = jest.fn(), uploading = false, uploadComplete = false, uploadError: any = null, downloadResults?: Partial<DownloadResultsState>, releaseCreated = false, releaseFailed = false) => {
+    const createStore = (hasUploadPermission = true, getUserCanUpload = jest.fn(), uploading = false, uploadComplete = false, uploadError: any = null, downloadResults?: Partial<DownloadResultsState>, releaseCreated = false, releaseFailed = false, ClearStatus = jest.fn()) => {
```

Comment:
Should camel-case `ClearStatus` for consistency here I think, even though it makes setting the value a little more verbose.